Dockerfile.fcos

```diff
@@ -0,0 +1,52 @@
+# Please ensure this file is kept in sync with Dockerfile.ocp
+FROM fedora:37 AS builder
+
+WORKDIR /tmp
+
+COPY prepare-efi.sh /bin/
+RUN prepare-efi.sh fedora
+
+FROM quay.io/centos/centos:stream8
```

Comment:
I strongly suggest to use stream9, all the packages used comes from cs9/rhel9 and this could break without notice
ironic upstream has removed support for python3.6 and CS8 since the beginning of the year

Reply:
Updated